src/python/foglamp/update_statistics_history.py

```diff
@@ -0,0 +1,139 @@
+"""
+The following piece of code takes the information found in the statistics table, and stores it's delta value 
+(statistics.value - statistics.prev_val) inside the statistics_history table. To complete this, SQLAlchemy will be 
+used to execute SELECT statements against statistics, and INSERT against the statistics_history table.  
+"""
+import sqlalchemy
+import time
+
+__author__ = "Ori Shadmon"
+__copyright__ = "Copyright (c) 2017 OSI Soft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+# Set variables for connecting to database
+_db_type = "postgres"
+_user = "foglamp"
+_db_user = "foglamp"
+_host = "192.168.0.182"
+_db = "foglamp"
+
+# Create Connection
+_ENGINE = sqlalchemy.create_engine('%s://%s:%s@%s/%s' % (_db_type, _db_user, _user, _host, _db),  pool_size=20,
+                                   max_overflow=0)
+_CONN = _ENGINE.connect()
```

Comment:
Globals like _CONN are initialized at module import time. This is not the time to do complex things like I/O because, if the guidelines are followed, there is no way to handle exceptions via import statements (it is possible to do this in Python but should be avoided for simplicity). Complex initialization should be performed in application logic. If you made this change and ran pylint against it, it would complain that you are using the 'global' keyword. Use a class attribute instead.